src/Resolver/IndexableResolver.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @author Peter Ukena <peter.ukena@brille24.de>
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusElasticsearchPlugin\Resolver;
+
+use Sylius\Component\Core\Model\ProductInterface;
+
+class IndexableResolver implements IndexableInterface
+{
+    /** {@inheritdoc} */
+    public function isProductIndexable(ProductInterface $product): bool
```

Comment:
With PHP7+ we don't use inheritdocs as it's no longer necessary. Could you remove it?